src/pages/HomePage.js

```diff
@@ -1,16 +1,20 @@
 import React from "react";
 import MostPopularServices from "../components/MostPopularServices";
+import SectionYourGovernment from "../sections/SectionYourGovernment";
 
 const HomePage = () => (
-  <div className="dg_section dark">
-    <div className="container">
-      <div className="row">
-        <div className="col">
-          <MostPopularServices />
+  <React.Fragment>
+    <div className="dg_section dark">
+      <div className="container">
+        <div className="row">
+          <div className="col">
+            <MostPopularServices />
```

Comment:
If the comment above about defining sections is true, I will adjust this component as part of this pr.

Reply:
This has been changed.